### ISSUE TITLE
Items disappear

### ISSUE THREAD

**Issue by DanSman** [dragotin/kraft#102]
From time to time when I create some quotes, invoices, etc. and then click "OK" and reopen them, items randomly disappear. 
This is usually the case when I change several items at the same time. 

This is done with sqlite. I will give it a try with mariadb.

**Comment by dragotin** [comment]
Do you remember if, if that happens, _reordering_ of items or _removal_ of items was involved? I have got reports about this bug, but was not yet able to reproduce it.

**Comment by DanSman** [comment]
I just tried that but nothing. 
I think it was reordering but I can't remember.

**Comment by DanSman** [comment]
It happened again. I copied a document and put a template at the beginning of the positions (approx 10 in total). Then I deleted the second position, but the first one is marked as deleted.
![items_dissappear](https://user-images.githubusercontent.com/28892085/114322693-b3813380-9b75-11eb-876c-99f91c6e6f9e.png)

This is reproducible.

The solution is to add the position at the top, press ok and open the document again. But I remember that sometimes another position just disappears as well (mostly the last position).
